sandbox.html

```diff
@@ -154,6 +156,16 @@
         }
       }
     }
+
+    function parseFilterVariables(filterString) {
+      return filterString.split(',').reduce((result, filterVarAndVal) => {
+        const [varName, varValue] = filterVarAndVal.split('=');
+        return {
+          ...result,
+          [varName]: varValue
+        };
+      }, {});
+    }
```

Comment:
You cannot pass number or Date values like this. Proposal: Use a text area where people enter JSON.
```
function parseFilterVariables (filterString) {
  return JSON.parse(filterString, (_, value) => {
    if (value === '{{currentDate}}') { // convenience for current date
      return new Date();
    }
    if (value.match?.(/....-..-..T..:..:..\....Z/)) {
      return new Date(value);
    }
    return value;
  });
}
```

Reply:
Done.